html/pfappserver/root/static.alt/src/views/Configuration/_components/AdminRoleView.vue

```diff
@@ -123,6 +123,7 @@ export default {
         if (this.id) {
           // existing
           this.$store.dispatch(`${this.storeName}/getAdminRole`, this.id).then(form => {
+            if (this.isClone) form.id = `${form.id}-${this.$i18n.t('copy')}`
```

Comment:
Is it possible to isolate this logic in a function instead of a full copy/paste ?

Reply:
@julsemaan not currently. There are long-term plans to move the business logic out of the templates and into the store modules. Until then, the individual templates are currently the only code-path that has access to both `isClone` and the API response.